diesel/src/query_source/joins.rs

```diff
@@ -153,9 +153,19 @@ where
 // {
 // }
 
-/// Indicates that two tables can be used together in a JOIN clause.
-/// Implementations of this trait will be generated for you automatically by
-/// the [association annotations](../associations/index.html) from codegen.
+/// Indicates that two tables can be joined without an explicit `ON` clause.
+///
+/// Implementations of this trait are generated by invoking [`joinable!`].
+/// Implementing this trait means that you can call
+/// `left_table.inner_join(right_table)`, without supplying the `ON` clause
+/// explicitly. To join two tables which do not implement this trait, you will
+/// need to call [`.on`].
```

Comment:
I think this sentence should state on which part of the query `on` should be called.

Reply:
Do you think that is sufficiently covered by the documentation for `.on`?